modules/nw-externalip-object.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-object_{context}"]
+= External IP address pool configuration
+
+The Network custom resource (CR) is defined by the `Network.config.openshift.io` CustomResourceDefinition.
+The configuration for external IP address blocks is defined by a Network custom resource (CR) named `cluster`.
+
+[IMPORTANT]
+====
+During cluster installation, the Cluster Version Operator (CVO) creates an object of the `Network.config.openshift.io` CR type named `cluster`.
+Creating any other CR objects of this type is not supported.
+====
+
+The following YAML describes the external IP configuration:
+
+.Network.config.openshift.io CR named `cluster`
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  externalIP:
+    autoAssignCIDRs: [] <1>
+    policy: <2>
+      ...
+----
+<1> Defines the IP address block in CIDR format that is available for automatic assignment of external IP addresses to a Service CR when `spec.type=LoadBalancer` is set.
+Only a single IP address range is allowed.
+
+<2> Defines restrictions on an IP address manually assigned to a Service CR.
+If no restrictions are defined, manually specifying the `spec.externalIPs` parameter in a Service CR is not allowed.
+By default, no restrictions are defined.
+
+The following YAML describes the parameters for the `policy` stanza:
+
+[id="policy-stanza_{context}"]
+.Network.config.openshift.io `policy` stanza
+[source,yaml]
+----
+policy:
+  allowedCIDRs: [] <1>
+  rejectedCIDRs: [] <2>
+----
+<1> A list of allowed IP address ranges in CIDR format.
+<2> A list of rejected IP address ranges in CIDR format.
+
+[id="example-external-ip-configuration_{context}"]
+== Example external IP pool configurations
+
+The following YAML describes the configuration of an external IP address block:
+
+.Example configuration with `spec.externalIP.autoAssignCIDRs` set
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  ...
+  externalIP:
+    autoAssignCIDRs:
+    - 192.168.132.254/29
+----
+
+The following example configures policy rules for allowed and rejected CIDR ranges.
+Any IP address that exists in both the allowed and rejected ranges is rejected.
+
+.Example configuration with `spec.externalIP.autoAssignCIDRs` set
```

Comment:
Line 57 is for spec.externalIP.autoAssignCIDRs set.
This line should be 'spec.externalIP.policy.allowedCIDRs and spec.externaIP.policy.rejectedCIDRs' set

Reply:
@weliang1, oops, thanks!